src/controllers/auth/logout.ts

```diff
@@ -0,0 +1,8 @@
+import { Express } from '../../types';
+
+export const logout: Express.MiddleWare = async (req, res) => {
+  const message = 'Logged out successfully';
+  return res.status(200).send({
+    message,
+  });
+};
```

Comment:
this method is just returning a 200 message - isn't there anything else that needs to be done to log the user out? are there any session cookies that we would want to delete?